scripts/install_swiftlint.sh

```diff
@@ -7,7 +7,7 @@
 set -e
 
 SWIFTLINT_PKG_PATH="/tmp/SwiftLint.pkg"
-SWIFTLINT_PKG_URL="https://github.com/realm/SwiftLint/releases/download/0.9.1/SwiftLint.pkg"
+SWIFTLINT_PKG_URL="https://github.com/realm/SwiftLint/releases/download/0.39.1/SwiftLint.pkg"
```

Comment:
There should be a way to get just the latest release. That would be a better option.